### PR TITLE
Practica 1

### PR DESCRIPTION
**Pull request by Tomas200019**
batalle mucho con el git bush 🌵

### CODE REVIEW

File: ago-dic-2020/Tomas O. Salazar/ExamenParcial1/libros.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+
+	public class libros {
```
Comment (aimiebell) [inline ago-dic-2020/Tomas O. Salazar/ExamenParcial1/libros.java:3]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

File: ene-jun-2021/TomásAntonioOrdazSalazar/dinosaurios.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package algoritmos;
+
+public class dinosaurios {
+	public static void main(String args[]) {
+		String a [ ]= {"Parasaurolophus" ,"Oviraptor" ,"Minmi" ,"Troodon", "Wannanosaurus"};
```
Comment (schiob) [inline ene-jun-2021/TomásAntonioOrdazSalazar/dinosaurios.java:5]: muy bien, solo por acá agarra el arreglo desde la terminal, para que pueda funcionar el algoritmo con diferentes entradas

**Review by schiob — APPROVED** [review]
Todo bien, solo checa lo de la entrada para la práctica 3 :smile:

**Comment by Tomas200019** [comment]
hice lo que pude profe

**Pull request closed without merging**